### PR TITLE
6to5 Sublime support blog post

### PR DESCRIPTION
**Pull request by zertosh**
I was going to use the gif in the README for https://github.com/6to5/6to5-sublime/ but they're really distracting. But if others disagree, I'm cool with switching to the gif or not having the comparison images at all.

### CODE REVIEW

File: _posts/2015-01-19-6to5-sublime.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title:  "6to5 Sublime support"
+author: Andres Suarez
+date:   2015-01-19 10:40:00
+categories: announcements
+share_text: "6to5 Sublime support"
+---
+
+We are pleased to announce <a href="https://github.com/6to5/6to5-sublime/" target="_blank">6to5-sublime</a>, a set of language syntax definitions for modern JavaScript with JSX extensions, as well as snippets for common React constructs (like lifecycle methods) in ES6. It's available as <a href="https://packagecontrol.io/packages/6to5" target="_blank">6to5</a> on <a href="https://packagecontrol.io" target="_blank">Package Control</a>. Eliminating barriers to ES6+ adoption is not just about having the most feature-ready transpiler – it's also about providing the right environment in which to write ES6+ JavaScript.
```
Comment (thejameskyle) [inline _posts/2015-01-19-6to5-sublime.md]: Could you get rid of the `target="_blank"`'s and just use the markdown syntax for links?

**Comment by thejameskyle** [comment]
I think GIFs would be nice. I'd like to see a couple smaller ones with captions.

**Comment by zertosh** [comment]
@thejameskyle wow sorry - total brain fart on not using markdown. I added the current gif so you could take a look. But when you say "smaller ones", what size do you mean exactly?

**Comment by thejameskyle** [comment]
In general I like these sorts of things to read as stories. So I would break it down like this:

- While sublime-react is an excellent project, we wanted more than it was giving us...
- Here's some issues with sublime-react...
- So we went ahead and created 6to5-sublime...
- [A bunch of gifs showing improvements over sublime-react...]
- You can set it up by doing the following...

**Comment by thejameskyle** [comment]
@zertosh I mean smaller ones that focus on individual improvements. (same height and width for each gif)

screen shot 2015-01-19 at 1 45 10 pm
screen shot 2015-01-19 at 1 44 40 pm
screen shot 2015-01-19 at 1 44 54 pm

**Comment by zertosh** [comment]
@thejameskyle The breakdown makes sense. I didn't really know what I was going for since the only other post is the "6to5 esnext". As for the smaller, yet more focused gifs, great idea! The existing ones were kinda rushed from code I had laying around. I need more time to make them. I'm going to close this PR and come back to it later in the week. Thanks for looking at it!

**Comment by thejameskyle** [comment]
Sure thing. You can leave this PR open if you want though and just push to it later.

**Comment by Dashed** [comment]
@zertosh Still working on that blog post? :smile:

**Comment by zertosh** [comment]
@Dashed Sigh. I wish I didn't have a day job :cry: But today is a snow day in NYC! Which also means my flight to SF for the React.js Conf got cancelled, so I'm going to miss the morning of the first day :(

**Comment by zertosh** [comment]
@thejameskyle Do you think these new screenshots are good enough for a blog post? https://github.com/babel/babel-sublime#screenshots

**Comment by thejameskyle** [comment]
They look great! I'd definitely like to see this blog post

**Comment by Dashed** [comment]
The screenshots + overview of what babel-sublime offers over alternatives is a huge improvement :+1:

**Pull request closed without merging**